automated/linux/ethernet/test-eth2.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
https://github.com/Linaro/test-definitions/blob/master/automated/linux/ethernet/ethernet.sh
I think we can try to avoid to reinvent the wheel